package.json

```diff
@@ -59,10 +62,29 @@
     "eslint-config-babel": "^1.0.1",
     "eslint-plugin-babel": "^3.3.0",
     "eslint-plugin-flow-vars": "^0.5.0",
-    "fs-extra": "^0.30.0",
-    "mocha": "^2.4.5",
+    "jest": "^19.0.2",
     "normalize-newline": "^2.1.0",
-    "nyc": "^10.1.2"
+    "pify": "^2.3.0",
+    "recursive-copy": "^2.0.6"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.js"
+    ],
+    "modulePathIgnorePatterns": [
+      "<rootDir>/test/fixtures"
```

Comment:
Jest does aggressive pre-caching of package.json files, apparently, and this was seriously harshing the mellow of a CI process when booting up the tests. So we ignore it. 😎